Auto update failing for frp

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [frp](https://github.com/termux/termux-packages/blob/7e068de3932af727c9572b675bd67754b939677c/packages/frp) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating frp [Current version: 0.66.0]
INFO: package being updated to 0.67.0.
Downloading https://github.com/fatedier/frp/archive/refs/tags/v0.67.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  504k    0  504k    0     0   519k      0 --:--:-- --:--:-- --:--:--  519k

You are about to commit these changes:

--------------------
diff --git a/packages/frp/build.sh b/packages/frp/build.sh
index 1d33ccaa4f..66c696f2f1 100644
--- a/packages/frp/build.sh
+++ b/packages/frp/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/fatedier/frp
 TERMUX_PKG_DESCRIPTION="A fast reverse proxy to expose a local server behind a NAT or firewall to the internet"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="2096779623 <admin@utermux.dev>"
-TERMUX_PKG_VERSION="0.66.0"
+TERMUX_PKG_VERSION="0.67.0"
 TERMUX_PKG_SRCURL=https://github.com/fatedier/frp/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=afe1aca9f6e7680a95652e8acf84aef4a74bcefe558b5b91270876066fff3019
+TERMUX_PKG_SHA256=18d0a35b965fab7e348aafc7b587847dd04ef2ef84822ed8fd5b9fe46b7ff6d7
 TERMUX_PKG_REPLACES="frpc, frps"
 TERMUX_PKG_BREAKS="frpc, frps"
 TERMUX_PKG_BUILD_IN_SRC=true
--------------------

bump(main/frp): 0.67.0


Not committing to Git!

INFO: Trying to build package.
INFO: Package frp exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building frp for arch aarch64...
::group::INFO: Fetching repo metadata
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100 13134  100 13134    0     0  17561      0 --:--:-- --:--:-- --:--:-- 17558
[packages-cf.termux.dev-apt-termux-main]: 100 13134  100 13134    0     0  17559      0 --:--:-- --:--:-- --:--:-- 17558
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100 13320  100 13320    0     0  18012      0 --:--:-- --:--:-- --:--:-- 18000
[packages-cf.termux.dev-apt-termux-root]: 100 13320  100 13320    0     0  18011      0 --:--:-- --:--:-- --:--:-- 18000
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100 13120  100 13120    0     0  18146      0 --:--:-- --:--:-- --:--:-- 18146
[packages-cf.termux.dev-apt-termux-x11]: 100 13120  100 13120    0     0  18144      0 --:--:-- --:--:-- --:--:-- 18146
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100   833  100   833    0     0   1139      0 --:--:-- --:--:-- --:--:--  1141
[packages-cf.termux.dev-apt-termux-main]: gpg: Signature made Sat 31 Jan 2026 05:56:54 AM UTC
[packages-cf.termux.dev-apt-termux-main]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-main]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-main]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100   833  100   833    0     0   1084      0 --:--:-- --:--:-- --:--:--  1084
[packages-cf.termux.dev-apt-termux-root]: eed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100   833  100   833    0     0   1131      0 --:--:-- --:--:-- --:--:--  1133
[packages-cf.termux.dev-apt-termux-root]: gpg: Signature made Sun 25 Jan 2026 06:17:46 PM UTC
[packages-cf.termux.dev-apt-termux-root]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-root]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: gpg: Signature made Sat 31 Jan 2026 01:56:32 AM UTC
[packages-cf.termux.dev-apt-termux-x11]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-x11]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
::endgroup::
Downloading frp@0.67.0 source from 'https://github.com/fatedier/frp/archive/refs/tags/v0.67.0.tar.gz' if necessary...
Downloading https://github.com/fatedier/frp/archive/refs/tags/v0.67.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  504k    0  504k    0     0   525k      0 --:--:-- --:--:-- --:--:--  525k
Applying patch: Makefile.patch
Downloading https://go.dev/dl/go1.25.6.linux-amd64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100    75  100    75    0     0    792      0 --:--:-- --:--:-- --:--:--   797

100 57.0M  100 57.0M    0     0  98.8M      0 --:--:-- --:--:-- --:--:-- 98.8M
env go build -trimpath -ldflags "-s -w" -tags frpc -o bin/frpc ./cmd/frpc
go: downloading github.com/rodaine/table v1.2.0
go: downloading github.com/spf13/cobra v1.8.0
go: downloading github.com/fatedier/golib v0.5.1
go: downloading github.com/fatedier/yamux v0.0.0-20250825093530-d0154be01cd6
go: downloading github.com/quic-go/quic-go v0.55.0
go: downloading github.com/samber/lo v1.47.0
go: downloading github.com/pelletier/go-toml/v2 v2.2.0
go: downloading github.com/spf13/pflag v1.0.5
go: downloading gopkg.in/ini.v1 v1.67.0
go: downloading k8s.io/apimachinery v0.28.8
go: downloading github.com/pion/stun/v2 v2.0.0
go: downloading golang.org/x/net v0.43.0
go: downloading golang.org/x/sync v0.16.0
go: downloading golang.org/x/time v0.5.0
go: downloading github.com/coreos/go-oidc/v3 v3.14.1
go: downloading golang.org/x/oauth2 v0.28.0
go: downloading github.com/gorilla/mux v1.8.1
go: downloading github.com/pires/go-proxyproto v0.7.0
go: downloading github.com/xtaci/kcp-go/v5 v5.6.13
go: downloading github.com/songgao/water v0.0.0-20200317203138-2b4b6d7c09d8
go: downloading github.com/vishvananda/netlink v1.3.0
go: downloading golang.zx2c4.com/wireguard v0.0.0-20231211153847-12269c276173
go: downloading golang.org/x/crypto v0.41.0
go: downloading github.com/Azure/go-ntlmssp v0.0.0-20221128193559-754e69321358
go: downloading golang.org/x/sys v0.35.0
go: downloading sigs.k8s.io/yaml v1.3.0
go: downloading k8s.io/utils v0.0.0-20230406110748-d93618cff8a2
go: downloading github.com/golang/snappy v0.0.4
go: downloading github.com/pion/dtls/v2 v2.2.7
go: downloading github.com/pion/transport/v3 v3.0.1
go: downloading github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5
go: downloading github.com/go-jose/go-jose/v4 v4.0.5
go: downloading golang.org/x/text v0.28.0
go: downloading github.com/klauspost/reedsolomon v1.12.0
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/templexxx/xorsimd v0.4.3
go: downloading github.com/tjfoc/gmsm v1.4.1
go: downloading sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading github.com/vishvananda/netns v0.0.4
go: downloading github.com/pion/logging v0.2.2
go: downloading github.com/pion/transport/v2 v2.2.1
go: downloading github.com/templexxx/cpu v0.1.1
go: downloading github.com/klauspost/cpuid/v2 v2.2.6
web/frpc/embed.go:9:12: pattern dist: no matching files found
make: *** [Makefile:38: frpc] Error 1
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occurred when I last tried to update at 2026-01-31 06:25:05 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/21539968120">21539968120</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/frp/build.sh
+++ b/packages/frp/build.sh
@@ -2,15 +2,28 @@ TERMUX_PKG_HOMEPAGE=https://github.com/fatedier/frp
 TERMUX_PKG_DESCRIPTION="A fast reverse proxy to expose a local server behind a NAT or firewall to the internet"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="2096779623 <admin@utermux.dev>"
-TERMUX_PKG_VERSION="0.66.0"
+TERMUX_PKG_VERSION="0.67.0"
 TERMUX_PKG_SRCURL=https://github.com/fatedier/frp/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=afe1aca9f6e7680a95652e8acf84aef4a74bcefe558b5b91270876066fff3019
+TERMUX_PKG_SHA256=18d0a35b965fab7e348aafc7b587847dd04ef2ef84822ed8fd5b9fe46b7ff6d7
 TERMUX_PKG_REPLACES="frpc, frps"
 TERMUX_PKG_BREAKS="frpc, frps"
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_AUTO_UPDATE=true
 
 termux_step_make() {
+	(
+		termux_setup_nodejs
+		unset CC CXX CFLAGS CXXFLAGS CPPFLAGS LDFLAGS AR AS CPP LD RANLIB READELF STRIP PREFIX PKGCONFIG PKG_CONFIG PKG_CONFIG_DIR PKG_CONFIG_LIBDIR
+		pushd web/frpc
+		npm install
+		npm run build
+		popd # web/frpc
+		pushd web/frps
+		npm install
+		npm run build
+		popd # web/frps
+	)
+
 	termux_setup_golang
 	make frpc
 	make frps

